examples/moving_horizon_estimation/mhe.py

```diff
@@ -124,10 +124,9 @@ def get_solver_options(solver):
         options.hessian_approximation = "limited-memory"
         options.limited_memory_max_history = 50
         options.set_maximum_iterations(5)
-        options.print_level = 0
+        options.set_print_level(0)
         options.tol = 1e-1
-        options.bound_frac = 1e-10
-        options.bound_push = 1e-10
+        options.set_initialization_options(1e-10)
```

Comment:
I really like that :)
That was a good idea to create a class to hold the options, thanks for thinking of that!